test/storage.test.ts

```diff
@@ -1,65 +1,43 @@
 import * as support from './support'
 import * as storage from '../lib/storage'
 import * as channel from '../lib/channel'
-
-import Web3 = require('web3')
-import Promise = require('bluebird')
 import Payment from '../lib/Payment'
-import mongo from '../lib/mongo'
 import BigNumber from '../lib/bignumber'
+import Engine from '../lib/engines/engine'
+import Web3 = require('web3')
+
 let expect = require('expect')
 
-const engineName = process.env.engineName || 'nedb'
+const engineName = process.env.ENGINE_NAME || 'nedb'
```

Comment:
This bug was preventing the storage tests from running against any engine other than `nedb`.